### DISCUSSION TITLE
Autofill without Ionic UI components

### DISCUSSION THREAD

**Discussion by freon27** [ionic-team/capacitor#7337] [Q&A]
Hi, 

I'm wondering whether it's possible to implement autofill without using `ion-input`?

We have a Capacitor app but we don't use the Ionic UI components.

The documentation seems to suggest that `ion-input` is needed but maybe there's a way around this?

Thanks
Nick

**Comment by jcesarmobile** [comment]
I think it should work, have you tried?

**Reply by freon27** [reply]
@jcesarmobile Not yet, I wanted to know whether it would be wasted time to investigate further. 

I'm also relatively new to Capacitor / Native and I don't totally understand what the trigger would be for the autofill to work in a normal input field. Is it just based on the autocomplete attributes?
